packages/Pill/src/Pill.styles.js

```diff
@@ -0,0 +1,76 @@
+import { css } from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+
+const PillColorStyles = {
```

Comment:
nit: `pill` lowercase, and also are we using color or colour @mikrotron 😄

Reply:
We use `color` everywhere :D, css uses `color` looool

Reply:
Yeah, `color` is pretty standard in our code (and pretty much everyone's).  That doesn't mean I like it, or that it's right, but it is consistent.